packages/app-studio-toolkit/package.json

```diff
@@ -22,8 +22,8 @@
     "ci": "npm-run-all clean compile coverage bundle package",
     "clean": "rimraf ./dist *.vsix",
     "compile": "tsc -p ./",
-    "test": "mocha",
-    "coverage": "nyc mocha",
+    "test": "mocha --timeout 5000",
```

Comment:
1. configurations for mocha should be in the mocharc (or however it is called)
2. setting high timeout for **all** tests is a bad practice.

Reply:
done for specific tests but for the whole scope